### PR TITLE
Small fixes update formatting

### PR DESCRIPTION
**Pull request by aljones15**
@JoshDunnDev this just lints your PR and adds a test credential with presentation fields.

credential-card-presentation-pr-1
credential-card-presentation-pr-2

### CODE REVIEW

**Comment by JoshDunnDev** [comment]
@aljones15 So the QR code should not be on the "Credential Details" view and only the photo and QR code should be on the "Presentation View". I believe the consolidation happens on the wallet side so that may be why it's not happening here.

**Comment by JoshDunnDev** [comment]
After testing this branch on the wallet, it still works correctly. We just may need to update the tests to reflect the same thing.

File: test/components/Home.vue [hunk 1]
```diff
@@ -69,6 +69,7 @@ import Logo from './Logo.vue';
 
 const issuer = 'did:v1:test:1234';
 
+// this makes the mores for testing more or less feature
```
Comment (gannan08) [inline test/components/Home.vue]:
```suggestion
// used for testing the truncation feature
```

**Comment by aljones15** [comment]
> After testing this branch on the wallet, it still works correctly. We just may need to update the tests to reflect the same thing.

@JoshDunnDev not sure I'm understanding that comment in this context. All this PR does is lint and add some jsdoc comments. should not change any functionality, just make it more apparent.

**Comment by JoshDunnDev** [comment]
> > After testing this branch on the wallet, it still works correctly. We just may need to update the tests to reflect the same thing.
> 
> @JoshDunnDev not sure I'm understanding that comment in this context. All this PR does is lint and add some jsdoc comments. should not change any functionality, just make it more apparent.

Alright that's fine.

**Review by JoshDunnDev — APPROVED** [review]

**Review by gannan08 — APPROVED** [review]

**Pull request merged**